server/controllers/api.py

```diff
@@ -133,13 +137,22 @@ def name_to_assign_id(name):
     if assgn:
         return assgn.id
 
+@cache.memoize(1000)
+def get_current_version(name):
+    version = models.Version.query.filter_by(name=name).one_or_none()
+    if version:
+        return version.current_version, version.download_link
+    return None, None
+
 class Resource(restful.Resource):
     version = 'v3'
     method_decorators = [authenticate, check_version]
     # applies to all inherited resources
 
+    def __repr__(self):
```

Comment:
This is so that flask-cache can memoize some routes that will be hit often/won't change.